pallets/validators-manager/src/mock.rs

```diff
@@ -74,6 +86,15 @@ const MOCK_T2_PUBLIC_KEY_BYTES: [u8; 32] = [
     151, 4, 182,
 ];
 
+fn hex_to_u8_array(hex: &str) -> [u8; 32] {
```

Comment:
Is this copied from some place? Could it be in some common place? 
(More like a nit-picking. Since this is a mock, it may be harder to refactor it to a common place than to maintain it as we go).